### DISCUSSION TITLE
Can I add already existing prefixes?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#171] [Q&A] [answered]
If for example I wanna launch a mod launcher for a steam game,then I would need to be able to add the prefix to bottles.
So does that work?

**Comment by mirkobrombin** [comment]
Currently our importer support only Lutris and POL

**Reply by ghost** [reply]
What does that mean?
I mean,do you mean that I can only add Lutris an POL Prefixes?

**Reply by mirkobrombin** [reply]
Yes

**Reply by ghost** [reply]
Will Bottles support adding prefixes in the future?
Cause I wanna play games with mod launchers.

**Reply by mirkobrombin** [reply]
Where do these prefixes you talk about come from? Steam with Proton, Lutris, POL, q4wine?

**Reply by ghost** [reply]
Steam with proton.

**Comment by mirkobrombin** [comment, marked as answer]
Proton prefixes are slightly different from common wine prefixes. We will probably allow them to be imported after Bottles v4 but we cannot guarantee that they will work properly.
